### PR TITLE
Implement message input styling

### PR DESCRIPTION
**Pull request by anqitu**

### CODE REVIEW

File: src/main/webapp/user-page.html [hunk 1]
```diff
@@ -125,19 +125,44 @@ <h6 class="font-weight-lighter text-muted" id="aboutme"></h6>
...
+              >
+                <i class="fas fa-map-marker-alt mr-1"></i>
+                Location
+              </button>
+              <button
+                type="submit"
+                id="submit-button"
+                class="btn btn-info btn-sm rounded-0 font-weight-bolder"
+              >
+                <i class="fas fa-podcast mr-1"></i>
```
Comment (amad-person) [inline src/main/webapp/user-page.html]: Can the icon for share be changed to https://fontawesome.com/icons/share-square?style=solid

File: src/main/webapp/user-page.html [hunk 2]
```diff
@@ -125,19 +125,44 @@ <h6 class="font-weight-lighter text-muted" id="aboutme"></h6>
...
+                <input
+                  type="file"
+                  name="image"
+                  accept="image/*"
+                  id="message-image-upload-input"
+                  style="display: none;"
+                />
+              </label>
+              <button
+                id="location-button"
```
Comment (amad-person) [inline src/main/webapp/user-page.html:149]: Can this button have a disabled attribute until the onclick function is implemented? Clicking it results in empty messages being added to the application:

Screenshot 2019-07-02 at 10 54 31 PM

**Review by amad-person — CHANGES_REQUESTED** [review]
Looks great, some minor changes needed

File: src/main/webapp/user-page.html (hunk 1 above)
Comment (anqitu) [reply src/main/webapp/user-page.html]: ok sure!

File: src/main/webapp/user-page.html (hunk 2 above)
Comment (anqitu) [reply src/main/webapp/user-page.html:149]: sure!

**Review by amad-person — APPROVED** [review]

**Pull request merged**